satpy/enhancements/__init__.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2017.
+
+# Author(s):
+
+#   Martin Raspaud <martin.raspaud@smhi.se>
+
+# This file is part of satpy.
+
+# satpy is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+
+# satpy is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License along with
+# satpy.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Enhancements."""
+
+import numpy as np
+
+
+def stretch(img, **kwargs):
+    """Perform stretch."""
+    img.stretch(**kwargs)
+
+
+def gamma(img, **kwargs):
+    """Perform gamma correction."""
+    img.gamma(**kwargs)
+
+
+def invert(img, *args):
```

Comment:
Just for consistency shouldn't these all have `*args, **kwargs`? Or do you not like that since they wouldn't be used.